.github/workflows/regression.yml

```diff
@@ -32,7 +32,9 @@ on:
   # Don't want to run this on each PR commit, but because GH doesn't allow specifying different required checks
   # for pull request and merge queue, we need to "run" it in pull request, but in the jobs we will just auto pass.
   pull_request:
-
+  # Also, this workflow runs every Monday at 5:00 AM UTC.
+  schedule:
+    - cron: "0 5 * * 1"
```

Comment:
I'm a little concerned that a weeks worth of commits may make bisecting regressions more difficult, but we can see. We could change this to be nightly now rather than in the future.

Reply:
Discussed offline, we will start with weekly runs. If this causes difficult investigations in the future, we will increase the frequency of this.